.rubocop.yml

```diff
@@ -57,6 +57,11 @@ Metrics/MethodLength:
   CountComments: false
   Max: 100
 
+Metrics/ModuleLength:
```

Comment:
Specifically ignored the `Metrics/ModuleLength` to prevent these errors from continuing to crop up. Test blocks can become intentionally long.